plugins/grpc/gateway/v2.14.0/remote-packages.patch

```diff
@@ -0,0 +1,233 @@
+diff --git a/internal/descriptor/registry.go b/internal/descriptor/registry.go
```

Comment:
Once we confirm this is working as intended and passes our internal tests, are we planning to merge this as a patch .. or do we wait to get this merged upstream with @johanbrandhorst help.

Reply:
I'm looking at the patch upstream but I admit it is a very specific use case, and if patching like this is an acceptable solution to you (at least in the short term) then I would prefer this patch stays here. I'm happy to consider upstreaming it if this turns into a big maintenance burden for you down the road. How does that sound?